app/javascript/modules/listingDetailsAside/MobileListingDetailsProcess.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { ListingDetailItem, t, Mobile } from "@bloom-housing/ui-components"
+import { RailsListing } from "../listings/SharedHelpers"
+import { ListingDetailsInfoSession } from "./ListingDetailsInfoSession"
+import { ListingDetailsProcess } from "./ListingDetailsProcess"
+import { isOpen } from "../../util/listingUtil"
+import { ListingDetailsLotteryResults } from "../listingDetailsLottery/ListingDetailsLotteryResults"
+import { ListingDetailsLotteryInfo } from "../listingDetailsLottery/LotteryDetailsLotteryInfo"
+import { ListingDetailsOpenHouses } from "./ListingDetailsOpenHouses"
+
+export interface ListingDetailsSidebarProps {
+  listing: RailsListing
+  imageSrc: string
+}
+
+export const MobileListingDetailsProcess = ({ listing, imageSrc }: ListingDetailsSidebarProps) => {
+  const isApplicationOpen = isOpen(listing)
+  return (
+    listing &&
+    isApplicationOpen && (
```

Comment:
same thing here, we could move the `isApplicationOpen` call up into `listings-detail`